src/Tools/Pagination/Paginator.php

```diff
@@ -38,6 +40,16 @@ class Paginator implements Countable, IteratorAggregate
     private readonly Query $query;
     private bool|null $useOutputWalkers = null;
     private int|null $count             = null;
+    /**
+     * @var bool The auto-detection of queries style was added a lot later to this class, and this
```

Comment:
```suggestion
     * The auto-detection of queries style was added a lot later to this class, and this
```

For a property phpdoc, the description is just the phpdoc description, not something attached to the tag (and then, we don't need `@var` at all as there is a native type).